lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -561,7 +561,7 @@ $(document).ready(function() {
             $('#gollum-editor-submit').attr('disabled', false)
             alert('Someone else has modified this page while you were editing it. Please store your version on disk outside of the browser, reload this page and reapply your modifications.');
```

Comment:
We could also move this error message to `app.rb`, and get rid of the `if...else` structure here (just showing a generic alert that displays the server-generated response in all cases).

Reply:
Fair enough. As long as the submit button is still disabled when one of these errors show up.

Reply:
Ah, I didn't notice that's unique behavior for the 412 case. I'll just leave it like this then!